playwright/pages/pageElements/rightSideDetailsSection.ts

```diff
@@ -185,6 +185,12 @@ export class RightSideDetailsPage extends BasePage {
     await this.waitForDOMToFullyLoad();
   }
 
+  async typeAndSelectCustomerEmail(customerEmail = "customer@example.com") {
+    await this.searchCustomerInput.fill(customerEmail);
+    await this.selectCustomerOption.locator(`text=${customerEmail}`).click();
+    await this.waitForDOMToFullyLoad();
```

Comment:
suggestion: I'd not recommend use `waitForDOMToFullyLoad` as it not working after initial page load, try to use build in pw feature to wait for element